docs/README.md

```diff
@@ -1,7 +1,132 @@
-# 🚧 WORK IN PROGRESS 🚧
+📢 Use this project, [contribute](https://github.com/vtex-apps/add-to-cart-button) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+
+# Product Context
+
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-0-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
+
+The Product Context app is responsible for exposing data regarding a certain product to all of it's children.
+
+## Usage example
+
+To use this app, be sure to add it to you app's `manifest.json` file:
+
+```json
+{
+  "dependencies": {
+    "vtex.product-context": "0.x"
+  }
+}
+```
+
+then you can import any of the exported components and hooks from the app. Here's an example of a component that render's the name of the product whose data is stored in the nearest `ProductContext`:
+
+```tsx
+import React, { FC } from 'react'
+import { useProduct } from 'vtex.product-context'
+
+const MyComponent: FC = () => {
+  const productContextValue = useProduct()
+
+  return (
+    <Fragment>
+      {productContextValue?.product?.productName}
+    </Fragment>
+  )
+}
+
+export default MyComponent
+```
```

Comment:
If we are going to document the usage in TypeScript I think we should make it clear that this file is a TypeScript file. People will copy and paste this code and will not work.